fluent-plugin-google-cloud.gemspec

```diff
@@ -23,7 +23,7 @@ eos
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '> 0.9'
   gem.add_runtime_dependency 'googleauth', '~> 0.4'
-  gem.add_runtime_dependency 'grpc', '~> 1.0'
+  gem.add_runtime_dependency 'grpc', '= 1.0.1'
```

Comment:
I prefer keeping the original version there and just adding the upper bound:
```
  gem.add_runtime_dependency 'grpc', '~> 1.0', '< 1.0.2'
```
Would that work?